pkg/storaged/dialog.jsx

```diff
@@ -843,8 +843,10 @@ export const SelectSpaces = (tag, title, options) => {
                                             ]}
                             />);
 
+                        const key = block || desc;
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22291-08fb19bd-20250807-084103-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)